test-at/src/test/resources/policies/ISocket-OParquet-operators.json

```diff
@@ -52,6 +52,17 @@
           ]
         }
       }
+    },
+    {
+      "name": "timestamp",
+      "order": 1,
+      "type": "DateTime",
+      "outputFields": [
+        {"name":"timestamp", "type": "long"}
+      ],
+      "configuration": {
+        "inputFormat": "autoGenerated"
+      }
```

Comment:
Changed all tests, we never use this field. The next commit --amend include this